content/en/monitors/faq/how-can-i-export-alert-history.md

```diff
@@ -3,46 +3,51 @@ title: Export Monitor Alerts to CSV
 kind: faq
 ---
 
-Download a history of monitor alerts through the [hourly monitor data][1], which generates a CSV for the past 6 months (182 days). This CSV is **not** live; it is updated once a week on Monday at 11:59AM UTC. 
+Download a history of monitor alerts through the [hourly monitor data][1], which generates a CSV for the past 6 months (182 days). This CSV is **not** live; it is updated once a week on Monday at 11:59AM UTC.
 
-**Note**: You need to be an administrator of your organization to access the CSV file.
+**Notes**:
 
-To fetch the CSV using curl, use the following:
-
-{{< tabs >}}
-{{% tab "US" %}}
-
-```shell
-api_key=<DATADOG_API_KEY>
-app_key = <DATADOG_APPLICATION_KEY>
+- This feature is US only.
```

Comment:
```suggestion
- This feature is only supported for the Datadog US site.
```

otherwise it sounds like it's available for us3/gov users located in the usa

Reply:
Good catch, thanks! Will update.